serverless/src/requeueOrder/handler.js

```diff
@@ -0,0 +1,92 @@
+import AWS from 'aws-sdk'
+
+import { getApplicationConfig } from '../../../sharedUtils/config'
+import { getSqsConfig } from '../util/aws/getSqsConfig'
+import { getDbConnection } from '../util/database/getDbConnection'
+
+// AWS SQS adapter
+let sqs
+
+/**
+ * Requeues an order onto an SQS queue for processing
+ * @param {Object} event Details about the HTTP request that it received
+ * @param {Object} context Methods and properties that provide information about the invocation, function, and execution environment
+ */
+const requeueOrder = async (event, context) => {
+  // https://stackoverflow.com/questions/49347210/why-aws-lambda-keeps-timing-out-when-using-knex-js
+  // eslint-disable-next-line no-param-reassign
+  context.callbackWaitsForEmptyEventLoop = false
+
+  const { defaultResponseHeaders } = getApplicationConfig()
+
+  const { body } = event
+  const { params = {} } = JSON.parse(body)
+
+  const { orderId } = params
+
+  if (sqs == null) {
+    sqs = new AWS.SQS(getSqsConfig())
```

Comment:
I think we are using NodeJS version 16.x (per serverless.yml), so you could consider changing
```js
if (sqs == null) {
    sqs = new AWS.SQS(getSqsConfig())
}
```
to
```js
sqs ??= new AWS.SQS(getSqsConfig())
```